plugin/evm/validators_cache_test.go

```diff
@@ -0,0 +1,59 @@
+package evm
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/flare-foundation/coreth/tests/mocks"
+	"github.com/flare-foundation/flare/ids"
+)
+
+func TestWithCacheSize(t *testing.T) {
+	wantCacheSize := uint(42)
+
+	cfg := CacheConfig{}
+	WithCacheSize(wantCacheSize)(&cfg)
+
+	assert.Equal(t, wantCacheSize, cfg.CacheSize)
+}
+
+type TestValidatorCache struct {
+	ByEpochFunc func(uint64) (map[ids.ShortID]uint64, error)
+}
+
+func (tv TestValidatorCache) ByEpoch(e uint64) (map[ids.ShortID]uint64, error) {
+	return tv.ByEpochFunc(e)
```

Comment:
Nitpick.

```suggestion
func (t TestValidatorCache) ByEpoch(e uint64) (map[ids.ShortID]uint64, error) {
	return t.ByEpochFunc(e)
```

Reply:
Here I meant `tv` standing for `test validator`, and now it seems to me that is could become `tvc` standing for `test validator cache`, so we can have also `tvn` - for test validator normalizer, etc.
Do you mean with this comment that this is inappropriate and should be simply `t`?

Reply:
Yes, method receivers should be a single letter in 99% of cases